features/minion/repository/src/main/resources/features.boot

```diff
@@ -1,5 +1,7 @@
 # Minion Default Features
 pax-war
+spring/4.2.9.RELEASE_1
```

Comment:
It looks like this did the trick for fixing the Minion containers, since the smoke tests are passing now.

Can we add a comment here, something along the lines of:
```
# Install specific versions of the spring and spring-jms features to
# reduce the computations needed by the bundle dependency resolver
```

It would also be nice if we could refer to ${springVersion} as defined in the root pom instead of hard coding the version here.

Reply:
Ok, I'll check.